spec/controllers/relationships_controller_spec.rb

```diff
@@ -0,0 +1,80 @@
+require 'spec_helper'
+
+describe RelationshipsController do
+  let(:ryan) {Fabricate(:user)}
+  let(:laura) {Fabricate(:user)}
+  let(:ranger) {Fabricate(:user)}
+  let(:relationship) {Fabricate(:relationship, follower_id: ryan.id, leader_id: laura.id)}
```

Comment:
I have seen it above - but you don't need to use follower_id: ryan.id (with the ids all the time). You have defined `follower` method on relationship model. So `let(:relationship) { Fabricate(:relationship, follower: ryan, leader: laura) }`. Also all hashes should have empty space at the beginning and at the end. You are not saving anything by cutting it and it is slightly more readable. [Check this out](https://github.com/bbatsov/ruby-style-guide#spaces-operators)